src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -76,6 +76,7 @@ struct uct_dc_mlx5_ep {
     uint16_t              flush_rkey_hi;
     uct_rc_fc_t           fc;
     uct_dc_mlx5_base_av_t av;
+    uint8_t               dci_stream_channel_id;
```

Comment:
1. dci_channel_index
2. do we really have to keep dci_channel_index? can't we take it from dci?

Reply:
ok, missed that in random scheme same dci used by multiple eps